### DISCUSSION TITLE
Hiding VAT on my Invoice

### DISCUSSION THREAD

**Discussion by bliepp** [kimai/kimai#4341] [Questions & Answers] [answered]
Hello there,

I recently started using Kimai and would like to make use of the basic invoice feature. My current job does not require me to pay VAT so I set it to 0 in my invoice template (obviously). The thing is the template creates a PDF, where the 0% VAT is shown, which I find kind of ugly. I would like to hide it completely. According to the [docs](https://www.kimai.org/documentation/invoices.html#global-variables) there's a template variable `${invoice.tax_hide}`, which appearantly is used by the [default pdf twig template](https://github.com/kimai/kimai/blob/7a5b12762afa53dd59749b218bb2d2acc8ead665/templates/invoice/renderer/default.pdf.twig#L128). But how do I actually set the boolean flag to "true"? According to the docs it only applies for tax = 0, which is the case for me.

bliepp

**Comment by kevinpapst** [comment, marked as answer]
You can't, only plugins can toggle that boolean flag. 

You can simply adjust the template and upload it under a new name.

**Reply by bliepp** [reply]
Ah okay, I see, thanks for the reply. I was searching like a maniac because I've seen this flag and simply couldn't find it anywhere in the UI. I guess I'm gonna leave it as it is, then.
